Cargo.toml

```diff
@@ -31,14 +31,14 @@ config = { workspace = true }
 futures = { workspace = true }
 handlebars = { workspace = true }
 json-patch = { workspace = true }
-k8s-openapi = { workspace = true, features = ["v1_28", "schemars"] }
+k8s-openapi = { workspace = true, features = ["v1_30", "schemars"] }
```

Comment:
I chose the lowest possible version for the new and updated version of `k8s-openapi`.